stubs/opencv-python/cv2/__init__.pyi

```diff
@@ -0,0 +1,6 @@
+from cv2 import data as data, gapi as gapi, mat_wrapper as mat_wrapper, misc as misc, utils as utils, version as version
+from cv2.cv2 import *
+
+__all__: list[str] = []
```

Comment:
This is a bit weird. `__all__` is empty, but there's nonetheless a bunch of stuff that should be considered re-exported from the module? 🧐

Reply:
`__all__` is also empty in the source code and at runtime.
If `__all__` is empty, should it be omitted?

Btw, cv2 adds a bunch of code and modules dynamically, not sure if that's why they have an empty `__all__` there..